### ISSUE TITLE
DebugToolbarMiddleware is missing from MIDDLEWARE_CLASSES

### ISSUE THREAD

**Issue by wilhelmy** [Metalab/mos#75]
```
$ ./manage.py migrate
SystemCheckError: System check identified some issues:

ERRORS:
?: debug_toolbar.middleware.DebugToolbarMiddleware is missing from MIDDLEWARE_CLASSES.
	HINT: Add debug_toolbar.middleware.DebugToolbarMiddleware to MIDDLEWARE_CLASSES.
```

**Comment by wilhelmy** [comment]
I fixed it now by adding `debug_toolbar.middleware.DebugToolbarMiddleware` to `MIDDLEWARE_CLASSES` in common.py, but I guess you don't actually want the debug toolbar in common.py? I'm not enough of a django/mos expert to understand the exact way the debug settings override the normal ones and it's not the task I set out to look at today, so I'll just report this bug :)

**Comment by trascen** [comment]
Adding it to the development settings should be the right thing to do here. Alternatively we could remove debug_toolbar...

**Comment by hex2a** [comment]
fixed in #83
